src/main/java/address/model/datatypes/ViewablePerson.java

```diff
@@ -0,0 +1,146 @@
+package address.model.datatypes;
+
+import javafx.beans.property.*;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+
+import java.time.LocalDate;
+
+/**
+ * Extends {@link Viewable} for the Person domain object.
+ * @see Viewable
+ */
+public class ViewablePerson extends Viewable<Person> implements ObservableViewablePerson {
```

Comment:
Does not extend `Person` because it should exclude `UniqueData` superclass guarantees.